walkthroughs/howto-k8s-cross-cluster/README.md

```diff
@@ -152,7 +152,7 @@ blue
 
 You can also test it using a simple curler pod, like so:
 ```
->>kubectl -n default run -it curler --image=tutum/curl /bin/bash
+>>kubectl --generator=run-pod/v1 run -i --tty curler --image=public.ecr.aws/k8m1l3p1/alpine/curler:latest
```

Comment:
Can we update the just the image instead of addition flag like generators ? Do we need them ?

Reply:
If you update just the image, you will get this warning:

`kubectl run --generator=deployment/apps.v1 is DEPRECATED and will be removed in a future version. Use kubectl run --generator=run-pod/v1 or kubectl create instead.`

It will work though.

Reply:
I see this warning for above command 
```
kubectl --generator=run-pod/v1 run -i --tty curler --image=public.ecr.aws/k8m1l3p1/alpine/curler:latest
Flag --generator has been deprecated, has no effect and will be removed in the future.
```
I think we should add --rm flag, good to clean up resources. If customer want they can decided not to.

Reply:
Yes, indeed. I was using out of date kubectl version. Lets go for:

`kubectl run -i --tty curler --image=public.ecr.aws/k8m1l3p1/alpine/curler:latest --rm`